dev-games/ogre/ogre-1.11.1.ebuild

```diff
@@ -0,0 +1,138 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+CMAKE_REMOVE_MODULES="yes"
+CMAKE_REMOVE_MODULES_LIST="FindFreetype FindDoxygen FindZLIB"
+
+inherit cmake-utils
+
+DESCRIPTION="Object-oriented Graphics Rendering Engine"
+HOMEPAGE="https://www.ogre3d.org/"
+SRC_URI="https://github.com/OGRECave/${PN}/archive/v${PV}.zip -> ${P}.zip"
+
+LICENSE="MIT public-domain"
+SLOT="0/1.11"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="beta-components +cache cg debug doc double-precision egl examples
+	+freeimage gles2 json ois openexr +opengl pch profile +resman-legacy
+	resman-pedantic resman-strict tools"
+
+REQUIRED_USE="
+	|| ( gles2 opengl )
+	^^ ( resman-legacy resman-pedantic resman-strict )
+	examples? ( ois )"
+
+RESTRICT="test" #139905
+
+RDEPEND="
+	dev-libs/zziplib
+	media-libs/freetype:2
+	x11-libs/libX11
+	x11-libs/libXaw
+	x11-libs/libXrandr
+	x11-libs/libXt
+	cg? ( media-gfx/nvidia-cg-toolkit )
+	egl? ( media-libs/mesa[egl] )
+	freeimage? ( media-libs/freeimage )
+	gles2? ( media-libs/mesa[gles2] )
+	json? ( dev-libs/rapidjson )
+	ois? ( dev-games/ois )
+	openexr? ( media-libs/openexr )
```

Comment:
this will likely need `:=`

Reply:
Oh, yes... openexr has subslots.
Fixed.